### PR TITLE
Add choco and homebrew package

### PR DESCRIPTION
**Pull request by michidk**

### CODE REVIEW

File: package/chocolatey/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿
+$ErrorActionPreference = 'Stop';
+
+$packageName = 'punktf'
+$destDir     = Join-Path $(Get-ToolsLocation) $packageName
+$url64       = 'https://github.com/neovim/neovim/releases/download/release/punktf.exe'
```
Comment (Shemnei) [inline package/chocolatey/tools/chocolateyinstall.ps1:6]: I think this url is wrong

File: package/chocolatey/tools/chocolateyinstall.ps1 [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+
+$packageArgs = @{
+    packageName   = $packageName
+    installerType = 'exe'
+    url64bit      = $url64
+
+    softwareName  = 'punktf*'
+
+    checksum64    = ''
```
Comment (Shemnei) [inline package/chocolatey/tools/chocolateyinstall.ps1:16]: I will make a release so you can fill this

File: package/homebrew/punktf-bin.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
+class PunktfBin < Formula
+  version '1.0.0-alpha'
+  desc "A cross-platform multi-target dotfiles manager"
+  homepage "https://github.com/Shemnei/punktf"
+
+  if OS.mac?
+      url "https://github.com/Shemnei/punktf/releases/download/#{version}/punktf-#{version}-x86_64-apple-darwin.tar.gz"
+      sha256 ""
+  elsif OS.linux?
+      url "https://github.com/Shemnei/punktf/releases/download/#{version}/punktf-#{version}-x86_64-unknown-linux-musl.tar.gz"
```
Comment (Shemnei) [inline package/homebrew/punktf-bin.rb:10]: We probably will only have a `x86_64-unknown-linux-gnu` build for now

File: package/homebrew/punktf-bin.rb [hunk 4]
```diff
@@ -0,0 +1,19 @@
+class PunktfBin < Formula
+  version '1.0.0-alpha'
+  desc "A cross-platform multi-target dotfiles manager"
+  homepage "https://github.com/Shemnei/punktf"
+
+  if OS.mac?
+      url "https://github.com/Shemnei/punktf/releases/download/#{version}/punktf-#{version}-x86_64-apple-darwin.tar.gz"
```
Comment (Shemnei) [inline package/homebrew/punktf-bin.rb:7]: I don't think we will have a apple build at the start. This should probably be an error for now?

File: package/chocolatey/README.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+Chocolatey Punktf package
+===
+
+## Release a new version
+
+1. Change url and checksum in `tools/chocolateyinstall.ps1`
+2. Adjust version number in `punktf.nuget`
```
Comment (Shemnei) [inline package/chocolatey/README.md:7]: Should `punktf.nuget` be renamed to `punktf.nuspec` ?

**Review by Shemnei — COMMENTED** [review]
Very nice effort to release this tool on chocolatey :+1:

File: package/homebrew/punktf-bin.rb (hunk 4 above)
Comment (michidk) [reply package/homebrew/punktf-bin.rb:7]: Why not?

File: package/chocolatey/tools/chocolateyinstall.ps1 (hunk 2 above)
Comment (michidk) [reply package/chocolatey/tools/chocolateyinstall.ps1:16]: Please add the checksums to the release description

File: package/chocolatey/tools/chocolateyinstall.ps1 (hunk 1 above)
Comment (michidk) [reply package/chocolatey/tools/chocolateyinstall.ps1:6]: yep, just a placeholder

File: package/chocolatey/README.md (hunk 5 above)
Comment (michidk) [reply package/chocolatey/README.md]: Yep, thx!

File: package/chocolatey/tools/chocolateyinstall.ps1 (hunk 2 above)
Comment (Shemnei) [reply package/chocolatey/tools/chocolateyinstall.ps1:16]: Will do

File: package/homebrew/punktf-bin.rb (hunk 4 above)
Comment (Shemnei) [reply package/homebrew/punktf-bin.rb:7]: We need to figure out cross-compiling and we also can't test the build afterwards

Comment (michidk) [reply package/homebrew/punktf-bin.rb]: dropped darwin support for now

**Comment by michidk** [comment]
This is still a draft PR until we have a pre-release so we can figure out URLs and checksums

File: package/chocolatey/punktf.nuspec [hunk 6]
```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>punktf</id>
+    <version>1.0.0-alpha</version>
```
Comment (Shemnei) [inline package/chocolatey/punktf.nuspec:5]: Version should be `1.0.0-alpha.1`

File: package/homebrew/punktf-bin.rb [hunk 7]
```diff
@@ -0,0 +1,16 @@
+class PunktfBin < Formula
+  version '1.0.0-alpha'
```
Comment (Shemnei) [inline package/homebrew/punktf-bin.rb:2]: Version should be `1.0.0-alpha.1`

Comment (Shemnei) [reply package/homebrew/punktf-bin.rb:2]: And target should be `x86_64-unknown-linux-gnu`

**Comment by michidk** [comment]
Basically finished and ready to go. @Shemnei please add AUR instructions to readme

**Comment by michidk** [comment]
Awesome!

**Pull request merged**